### PR TITLE
[bitnami/mysql] Chart standardised

### PR DESCRIPTION
**Pull request by migruiz4**
**Description of the change**

This PR standardizes the bitnami/mysql chart to be in line with the rest of the catalog.

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the values.yaml and added to the README.md using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/mysql/templates/secrets.yaml [hunk 1]
```diff
@@ -13,9 +13,9 @@ metadata:
   {{- end }}
 type: Opaque
 data:
-  mysql-root-password: {{ include "mysql.root.password" . | b64enc | quote }}
-  mysql-password: {{ include "mysql.password" . | b64enc | quote }}
+  mysql-root-password: {{ include "common.secrets.passwords.manage" (dict "secret" (include "common.names.fullname" .) "key" "mysql-root-password" "length" 10 "providedValues" (list "auth.rootPassword") "context" $) }}
```
Comment (fmulero) [inline bitnami/mysql/templates/secrets.yaml:16]: Great! With this change we can read secrets during upgrades and we don't need to set the value `root.password`. Could your review NOTES.txt file and change validations/messages about this topic? An example (Sorry I can't comment in GH this part):
```console
To upgrade this helm chart:

  1. Obtain the password as described on the 'Administrator credentials' section and set the 'root.password' parameter as shown below:

      ROOT_PASSWORD=$(kubectl get secret --namespace default mysql -o jsonpath="{.data.mysql-root-password}" | base64 --decode)
      helm upgrade --namespace default mysql bitnami/mysql --set auth.rootPassword=$ROOT_PASSWORD
```

**Review by fmulero — CHANGES_REQUESTED** [review]
Great @migruiz4! I have some little suggestions, Could you have a look? Thanks

File: bitnami/mysql/templates/primary/svc.yaml [hunk 2]
```diff
@@ -17,25 +17,36 @@ metadata:
...
+  {{- if and (eq .Values.primary.service.type "LoadBalancer") (not (empty .Values.primary.service.loadBalancerSourceRanges)) }}
+  loadBalancerSourceRanges: {{ .Values.primary.service.loadBalancerSourceRanges }}
+  {{- end }}
+  {{- if and (eq .Values.primary.service.type "LoadBalancer") (not (empty .Values.primary.service.loadBalancerIP)) }}
+  loadBalancerIP: {{ .Values.primary.service.loadBalancerIP }}
   {{- end }}
   ports:
     - name: mysql
-      port: {{ .Values.primary.service.port }}
+      port: {{ .Values.primary.service.ports.mysql }}
```
Comment (fmulero) [inline bitnami/mysql/templates/primary/svc.yaml:40]: Remember to change `runtime_parameters` in VIB

Comment (migruiz4) [reply bitnami/mysql/templates/primary/svc.yaml:40]: MySQL does not have VIB `runtime_parameters` yet

Comment (fmulero) [reply bitnami/mysql/templates/primary/svc.yaml:40]: 🤦  my fault

**Review by fmulero — DISMISSED** [review]
LGTM!

**Pull request merged**